### PR TITLE
Tiny clean up.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: op-guide/build.md [hunk 1]
```diff
@@ -21,5 +21,5 @@ install the missing ones automatically.
 
 ## Building and installing TiDB components
 
-You can use the [build script](../scripts/build.sh) to build and install TiDB components in the bin` directory.
+You can use the [build script](../scripts/build.sh) to build and install TiDB components in the bin directory.
```
Comment (siddontang) [inline op-guide/build.md]: Use "\`bin\`"

**Comment by qiuyesuifeng** [comment]
PTAL

**Comment by shenli** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**